app/controllers/trips_controller.rb

```diff
@@ -0,0 +1,46 @@
+class TripsController < ApplicationController
+  before_action :set_trip, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @trips = Trip.all
+  end
+
+  def new
+    @trip = Trip.new
+  end
+
+  def create
+    @trip = Trip.new(trip_params)
```

Comment:
You can create a new Trip with Users with devise right now like current_user.trip.build :)